### PR TITLE
Point 2

### PR DESCRIPTION
**Pull request by f00se**
Point number 2. I will wait for feedback.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -2,4 +2,13 @@ class User < ApplicationRecord
...
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :rememberable, :trackable
+
+  validates :email, presence: true
+  validates :password, presence: true
+
+  enum role: [ :regular, :admin ]
+
+  def role?(test_role)
+    self.role == test_role.to_s
```
Comment (f00se) [inline app/models/user.rb]: I leave "self" to better know what's going on.

Comment (Exelord) [inline app/models/user.rb]: Ok, but in this context is not necessary

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -4,4 +4,12 @@
   it { is_expected.to validate_presence_of(:email) }
 
   it { is_expected.to validate_presence_of(:password) }
+
+  it "has role attribute" do
+    expect(subject.attributes).to include('role')
+  end
+
+  it "has default role regular" do
```
Comment (Exelord) [inline spec/models/user_spec.rb]: There is a missing test for an `admin` role.
Try to mock a user with regular role, then update the user with admin and check if correct role integer has been set

**Comment by f00se** [comment]
I will decide tomorrow when I will be working on next stage.

File: spec/models/user_spec.rb (hunk 2 above)
Comment (f00se) [inline spec/models/user_spec.rb]: fixed
`it 'has admin option available' do
    subject.role = "admin"
    expect(subject).to have_attributes(role: 'admin')
end`

**Pull request merged**